src/diffusers/pipelines/controlnet/pipeline_controlnet.py

```diff
@@ -616,33 +617,50 @@ def check_inputs(
         is_compiled = hasattr(F, "scaled_dot_product_attention") and isinstance(
             self.controlnet, torch._dynamo.eval_frame.OptimizedModule
         )
-        if (
```

Comment:
can we keep the code change minimum here? I don't think we need to change the code completely - maybe fit the modified logit into the original structure?

Reply:
Hi, sorry for a late reply. Now it seems to have minimal changes compared to the original code.